test/core/util/run_with_poller.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+GRPC_POLL_STRATEGY=$1 $2
```

Comment:
Wouldn't you want something like this instead ?

```
export GRPC_POLL_STRATEGY=$1
shift
$@
```

This way you could call run_with_poller.sh with arguments to whichever command you want to run.

Reply:
Note this is only for future-proofing this, not in the present state.